tools/SourceKit/lib/SwiftLang/CodeCompletionOrganizer.cpp

```diff
@@ -1162,24 +1153,28 @@ Completion *CompletionBuilder::finish() {
     const ContextFreeCodeCompletionResult &contextFreeBase =
         base.getContextFreeResult();
 
+    CodeCompletionString *newStr = contextFreeBase.getCompletionString();
+    NullTerminatedStringRef filterName = contextFreeBase.getFilterName();
```

Comment:
I think it would avoid confusion if either both of these variables start with with `new` or neither of them. I would suggest either
- `completionString` and `filterName` or
- `newCompletionString` and `newFilterName` (I think I have a slight preference for this one)